From 083f26aef3344aefa603b094285f0c28b6f0b04b Mon Sep 17 00:00:00 2001
From: Ken Tossell <ken@tossell.net>
Date: Sun, 11 Aug 2013 21:33:03 -0700
Subject: [PATCH 4/5] Added install rule for package.xml

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0132126..bfdaedf 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -32,3 +32,6 @@ install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/include/utlist.h"
 install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/include/libuvc/libuvc.h"
   DESTINATION include/libuvc)
 
+# Install catkin package.xml
+install(FILES package.xml DESTINATION share/libuvc)
+
-- 
1.8.4

